src/defaultStore/state.js

```diff
@@ -1,4 +1,9 @@
 export default function(options) {
+
+    if (typeof window.__ENV__ === 'undefined') {
```

Comment:
Afterthought, this definition could be moved to test specific environment to avoid unnecessary change here.